apps/front/src/components/job-postings/job-posting-card.svelte

```diff
@@ -0,0 +1,126 @@
+<style>
+    @media (min-width: 768px) {
+        a .container {
+            transition: top ease 250ms;
+        }
+        a:hover .container {
+            top: -0.3125rem;
+        }
+        a .logo {
+            transition: top ease 250ms 75ms;
+        }
+        a:hover .logo {
+            top: 0;
+        }
+    }
+</style>
+
+<script>
+    import { link } from 'svelte-routing';
+
+    export let jobPosting;
+    jobPosting = {
+        ...jobPosting,
+        identifier: 'a122edec-5580-4a93-aff7-fc18b41e4c37',
+        jobLocationType: 'Télétravail',
+        baseSalary: '30k - 34k',
+        skills: [],
+    };
+</script>
+
+<a
+    href="/jobs/{jobPosting.id}"
+    use:link
+    class="container flex relative flex-col my-5 mt-2-5 w-full
+    md:max-w-screen-sm max-w-sm md:ml-15"
+>
+
+    <div
+        class="logo absolute -top-2-5 left-5 md:top-2-5 md:-left-15 bg-white
+        rounded-lg flex shadow-xl w-15 h-15 md:w-25 md:h-25 p-2-5 flex flex-row
+        justify-center items-center z-10"
+    >
+        <img
+            class="max-w-full max-h-full"
+            src="{jobPosting.hiringOrganization.image}"
```

Comment:
tu peux virer les guillemets
```suggestion
            src={jobPosting.hiringOrganization.image}
```